TypeCobol/Compiler/Diagnostics/TypeCobolChecker.cs

```diff
@@ -728,9 +731,9 @@ private static void CheckParameter([NotNull] ParameterDescriptionEntry parameter
             {
                 foreach (var condition in parameter.DataConditions)
                 {
-                    if (condition.LevelNumber?.Value != 88)
+                    if (condition.LevelNumber?.Value != 88 && condition.Name != null)
```

Comment:
I think we should not change the condition in the test here but only the error message. So you can handle the case of null name directly in the error message like `"Condition parameter \"" + (condition.Name ?? "<missing name>") + "\" must be level 88."` or add a specific test for the null name before this one and create a new diagnostic just for the null name case.
To be discussed with @smedilol.

Reply:
Remove in grammar possibility of condition in procedure parameter.
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/77d03737d977c6907c9a5d2deea5d19ffd2cd74f

Reply:
Can you also remove the related C# code ?

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/39f4a7a95c122ab7a4af7c18a622d1a8679e1d5b